recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: wntr
+  version: "0.2.0"
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/3c8a669d50e77e872e9e6dd2c391a9fa83c34ecd.zip
+  sha256: 5d33a18f7cee0f76ca7a99074243a5019ae70f76ccb84dc44be54106eb758589
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - python
+    - pip
+    - numpy
+    - networkx
+    - pandas
+    - scipy
+    - matplotlib
+    - plotly
+    - nose
```

Comment:
do you really need ``nose`` to build the package or just to test it?
If is just to test it you can remove ``nose`` from the build requirements and also from host

Reply:
Okay, thank. I will remove it.